CONTRIBUTING.md

```diff
@@ -34,17 +34,17 @@ Some great guidelines can be found [here](https://wiki.openstack.org/wiki/GitCom
 ## Releases
 
 We try to stick to semantic versioning and our releases are automated. Release is created by assigning a keyword (in a
-way similar to travis [`[ci skip]`](https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build)) to a 
-commit with merge request. Available keywords are (square brackets are important!):
+way similar to circle ci keyword [`[ci skip]`](https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build))
```

Comment:
nit: we should probably link to circle ci docs

Reply:
Yea, things to fix in the skeleton repo :)